### PR TITLE
fix(std/test): Ensures time tests run in UTC

### PR DESCRIPTION
**Pull request by j4qfrost**
This sets the TZ to UTC to resolve flakiness of format and weekOfYear tests in std. I also added parentheses to python tool prints for compatibility with python3.

### CODE REVIEW

File: tools/hash_benchmark.py [hunk 1]
```diff
@@ -34,12 +34,12 @@ def run_benchmark(deno_exe, method, input_file):
         (out, _) = p.communicate()
 
         elapsed = out.split(':')[1].strip()
-        print "[{}] {}".format(alg, elapsed)
+        print("[{}] {}".format(alg, elapsed))
```
Comment (kitsonk) [inline tools/hash_benchmark.py:37]: Is there a reason for these changes?  They seem to be unrelated to the intent of this PR.

Comment (j4qfrost) [reply tools/hash_benchmark.py:37]: I added those changes in order to run with python 3 with my environment. They shouldn't be invasive towards the old workflow since it's compatible with python 2 also. Should this be in a separate PR?

Comment (bartlomieju) [reply tools/hash_benchmark.py:37]: @j4qfrost yes, change to Python code should be done in a separate PR.

File: cli/tests/integration_tests.rs [hunk 2]
```diff
@@ -9,6 +9,7 @@ use test_util as util;
 
 #[test]
 fn std_tests() {
+  std::env::set_var("TZ", "UTC");
```
Comment (bartlomieju) [inline cli/tests/integration_tests.rs:12]: This sets `TZ` env variable in the current env - that's not desirable, instead use `.env()` on the subprocess like for `DENO_DIR` below

**Review by bartlomieju — CHANGES_REQUESTED** [review]

**Comment by j4qfrost** [comment]
Accidentally closed this.

File: tools/hash_benchmark.py (hunk 1 above)
Comment (j4qfrost) [reply tools/hash_benchmark.py]: https://github.com/denoland/deno/pull/7779

File: cli/tests/integration_tests.rs [hunk 3]
```diff
@@ -13,6 +13,7 @@ fn std_tests() {
   let std_config = util::root_path().join("std/tsconfig_test.json");
   let status = util::deno_cmd()
     .env("DENO_DIR", dir.path())
+    .env("TZ", "UTC")
```
Comment (j4qfrost) [inline cli/tests/integration_tests.rs:16]: @bartlomieju done.

**Comment by bartlomieju** [comment]
Closing in favor of #8131
